accumulator/pollard.go

```diff
@@ -209,6 +208,23 @@ func (p *Pollard) rem2(dels []uint64) error {
 		}
 	}
 
+	// All the leaves to be deleted should be set to be not remembered.
+	// TODO there's probably a better way to do this than calling readPos
+	// a whole lot.
+	for _, del := range dels {
```

Comment:
it would probably be better to set remember=false when swapping the leaves

Reply:
I did try to get that to work but `TestCache()` kept failing and I bet there's a bunch of edge cases to handle. I'm leaning more towards just getting this in and then working on setting remember to false during swaps since that's an optimization.